apps/server/src/modules/files-storage/entity/filerecord.entity.spec.ts

```diff
@@ -113,4 +118,21 @@ describe('FileRecord Entity', () => {
 			expect(securityCheck.requestToken).toEqual(securityCheck.requestToken);
 		});
 	});
+
+	describe('setName is called', () => {
```

Comment:
I don't like this naming of tests with "is called". "FileRecord Entity setName is called should set name" doesn't make any sense as a sentence, whereas "FileRecord Entity setName should set name" is an understandable sentence.

Reply:
It's new to me that it should build up as a sentence. After discussion with @CeEv our understanding is, that the describe block is the description of the scenario. And in that manner we have the call of the function as the "outer" scenario. The inner describe blocks add further details of the scenario the test is executed in. As all tests in files storage are structured that way, it's more of a general discussion or knowledge sharing than something we will change in this PR.